tests/test_mqtt_block.py

```diff
@@ -1,19 +1,75 @@
+from unittest.mock import patch, MagicMock
 from nio.block.terminals import DEFAULT_TERMINAL
 from nio.signal.base import Signal
 from nio.testing.block_test_case import NIOBlockTestCase
-from ..example_block import Example
+from ..mqtt_publish_block import MqttPublish
+from ..mqtt_subscribe_block import MqttSubscribe
 
+import paho.mqtt.client as mqtt
 
-class TestExample(NIOBlockTestCase):
+class TestMqtt(NIOBlockTestCase):
 
-    def test_process_signals(self):
-        """Signals pass through block unmodified."""
-        blk = Example()
-        self.configure_block(blk, {})
-        blk.start()
-        blk.process_signals([Signal({"hello": "n.io"})])
-        blk.stop()
+    def test_mqtt_subscribe(self):
+        """Test that block is configured/connected correctly and that it
```

Comment:
Take a look at pep-257. I believe multi-line doc strings should still only have a one line "summary" line. Maybe just "Block notifies signals on client messages".

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings